api/search.go

```diff
@@ -167,6 +170,14 @@ func CreateRequests(w http.ResponseWriter, req *http.Request, validator QueryPar
 		Now:       time.Now().UTC().Format(time.RFC3339),
 	}
 
+	if nlpCritiria != nil && nlpCritiria.UseCategory {
+		reqSearch.NlpCategories = nlpCritiria.Categories
+	}
+
+	if nlpCritiria != nil && nlpCritiria.UseSubdivision {
+		reqSearch.NlpSubdivisionWords = nlpCritiria.SubdivisionWords
+	}
```

Comment:
We're adding `Nlp*` to the `reqSearch` struct, but is it actually being subsequently used? In other words, does it get put into part of the ES templated query to affect the returned data. At the least, running locally, it should be possible to see the impact of including a category term on the results when hitting the API with `curl` (and it's connected to a populated test ES instance).

As an example, tracking through the approach for [PopulationTypes](https://github.com/ONSdigital/dp-search-api/blob/d00812870a762b255f413a09b402a63526ba3fb6/query/templates/search/v710/populationTypeFilters.tmpl#L4) (and the ES NLP changes in the templates folder from the alpha), you should see the follow-on step -- or maybe I'm missing something else in this PR?